exonum-java-binding/core/rust/src/testkit/mod.rs

```diff
@@ -157,7 +157,7 @@ pub extern "system" fn Java_com_exonum_binding_testkit_TestKit_nativeCreateBlock
             let serialized_tx: jbyteArray = serialized_tx_object.as_obj().into_inner();
             let serialized_tx = env.convert_byte_array(serialized_tx)?;
             let transaction: Signed<RawTransaction> =
-                StorageValue::from_bytes(serialized_tx.into());
+                BinaryValue::from_bytes(serialized_tx.into()).unwrap(); // TODO: rewrite
```

Comment:
Why, how, Jira ref?

Reply:
I think that just `.unwrap()` is enough here. The cause of the error, if any, will be printed in logs.